docs/CloudPermissionsForScalarDLOnAKS.md

```diff
@@ -0,0 +1,80 @@
+# Cloud Permissions for Creating Scalar DL on Azure AKS
+
+This guide explains what permissions are required to deploy Scalar DL on Azure AKS.
+This guide assumes that you create the environment on the basis of [manual deployment guide for Azure](./ManualDeploymentGuideScalarDLOnAzure-1.md).
+From the security perspective, it is better to give users limited permissions to protect the environment from unwanted activities.
```

Comment:
Are the permissions of the following URL not sufficient? (just for aks)
https://docs.microsoft.com/en-us/azure/aks/concepts-identity

Reply:
The above document specifies the minimum permission for Scalar DL deployment on AKS. 

I think we do not need all permissions specified in the above URL.